### ISSUE TITLE
11.什么是响应式网页?

### ISSUE THREAD

**Issue by webVueBlog** [webVueBlog/interview-answe#11]
通过 `CSS3 Media Query` 实现响应式 `Web` 设计 

响应式 `Web` 设计(`Responsive Web design`)的理念是，页面的设计与开发应当根据用户行为以及设备环境(系统平台、屏幕尺寸、屏幕定向等)进行相应的响应和调整。

具体的实践方式由多方面组成，包括弹性网格和布局、图片、`CSS media query` 的使用等。

无论用户正在使用笔记本还是 `iPad`，我们的页面都应该能够自动切换分辨 率、图片尺寸及相关脚本功能等，以适应不同设备。

换句话说，页面应该有能力去自动响应用户的设备环境。这样，我们就可以不必为不断到来的新设备做专门的版本设计和开发了。

**Comment by webVueBlog** [comment]
Implement responsive Web design through CSS3 Media Query

The idea of Responsive Web design is that the design and development of pages should be Responsive and adjusted according to user behavior and the device environment (system platform, screen size, screen orientation, etc.).

The specific practices are composed of many aspects, including elastic grids and layouts, images, the use of CSS media queries, and so on.

Whether the user is using a laptop or an iPad, our pages should be able to automatically switch resolution, picture size, and related scripting to suit different devices.

In other words, the page should have the ability to automatically respond to the user's device environment. That way, we don't have to design and develop specific versions of the new devices that keep coming.
